src/main/java/sg/ncl/MainController.java

```diff
@@ -75,6 +77,9 @@
     @Autowired
     private RestTemplate restTemplate;
 
+    @Inject
+    private ObjectMapper objectMapper;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use constructor injection for this field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3306)